Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
             targets: ["MapboxCoreMapsWrapper"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/mapbox/mapbox-common-ios.git", from: "24.7.0-beta.1"),
+        .package(url: "https://github.com/mapbox/mapbox-common-ios.git", from: "11.7.0-beta.2"),
```

Comment:
Something goes wrong here @evil159 @Kylmakalle 
Version of common is incorrect

Reply:
ACK